cmd/fs-v1-multipart.go

```diff
@@ -233,12 +236,14 @@ func (fs fsObjects) newMultipartUpload(bucket string, object string, meta map[st
 		fsMeta.Meta = meta
 	}
 
-	// get a random ID for lock instrumentation.
-	opsID := getOpsID()
-
-	// This lock needs to be held for any changes to the directory contents of ".minio.sys/multipart/object/"
-	nsMutex.Lock(minioMetaBucket, pathJoin(mpartMetaPrefix, bucket, object), opsID)
-	defer nsMutex.Unlock(minioMetaBucket, pathJoin(mpartMetaPrefix, bucket, object), opsID)
+	// This lock needs to be held for any changes to the directory
+	// contents of ".minio.sys/multipart/object/"
+	objectMPartPathLock := nsMutex.NewNSLock(
+		minioMetaBucket,
```

Comment:
This type of indentation is not correct, keep the 'minioMetaBucket' close to 'NewNSLock(' and then rest of the others can come in new line.